//
// Academic License - for use in teaching, academic research, and meeting
// course requirements at degree granting institutions only.  Not for
// government, commercial, or other organizational use.
//
// abcdef_O_VP.h
//
// Code generation for function 'abcdef_O_VP'
//

#ifndef ABCDEF_O_VP_H
#define ABCDEF_O_VP_H

// Include files
#include "rtwtypes.h"
#include <cstddef>
#include <cstdlib>

// Function Declarations
void abcdef_O_VP(double P_init, double V_init, double A_init, double P_wayp,
                 double V_wayp, double V_max, double A_max, double A_min,
                 double J_max, double J_min, double t[7]);

#endif
// End of code generation (abcdef_O_VP.h)
